test/src/signatures.test.ts

```diff
@@ -63,13 +68,19 @@ describe('signatures', () => {
         challengeDuration: '0x5',
       };
       const hashedState = hashState(state);
-      const signature = splitSignature(await wallet.signMessage(arrayify(hashedState)));
-      expect(await signChallengeMessage([{state, signature}], wallet.privateKey)).toMatchObject({
-        r: expect.any(String),
-        s: expect.any(String),
-        v: expect.any(Number),
-      });
+      const stateSignature = splitSignature(await wallet.signMessage(arrayify(hashedState)));
+      const largestTurnNum = toHex(1);
+      const channelId = getChannelId(channel);
+      const challengeHash = hashChallengeMessage({largestTurnNum, channelId});
+      const challengeMessageSignature = splitSignature(
+        await wallet.signMessage(arrayify(challengeHash)),
```

Comment:
Why not use the web3 wallet here and make this test `sync`?

Reply:
This is partially a sanity check that the way we're using `web3-eth-accounts` is generating the same signature as before.  Now that it passes I'd feel comfortable switching it out :P